acceptance/steps/register.go

```diff
@@ -66,6 +66,7 @@ func init() {
 	// General cluster scenario steps
 	framework.RegisterStep(`^service "([^"]*)" has named port "([^"]*)" with value (\d+)$`, checkServiceWithPort)
 	framework.RegisterStep(`^service "([^"]*)" should have named port "([^"]*)" with value (\d+)$`, checkServiceWithPort)
+	framework.RegisterStep(`^rpk profile is configured in "([^"]*)" cluster$`, checkRPKCommands)
```

Comment:
I'm a bit confused here. These tests run against the v2 operator, which doesn't configure an rpk profile nor are there any changes being made to the v2 operator. Is this just to match the new tests added to v1?